user/frr/files/frr

```diff
@@ -0,0 +1,4 @@
+type = scripted
+command = /usr/lib/frr/frrinit.sh start
+stop-command = /usr/lib/frr/frrinit.sh stop
+depends-on = network.target
```

Comment:
`depends-on: network.target`

also needs a "proper" target dependency like local.target, network.target on its own is not sufficient